htp/htp_response.c

```diff
@@ -1070,21 +1070,49 @@ size_t htp_connp_res_data_consumed(htp_connp_t *connp) {
 }
 
 htp_status_t htp_connp_RES_FINALIZE(htp_connp_t *connp) {
-    int bytes_left = connp->out_current_len - connp->out_current_read_offset;
-    unsigned char * data = connp->out_current_data + connp->out_current_read_offset;
+    if (connp->out_status != HTP_STREAM_CLOSED) {
+        OUT_PEEK_NEXT(connp);
+        if (connp->out_next_byte == -1) {
+            return htp_tx_state_response_complete_ex(connp->out_tx, 0);
```

Comment:
This does not seem to be always the right thing to do, but is needed by unit tests

Reply:
When would this not be correct and why?

Reply:
When the end of a packet is `LF` but the next packet is not the start of a new request for instance.

For instance, in the private example you shared with me, is packet 7 a new response or continuation body of the previous response ?

Reply:
And this may not be correct because we want to interpret this junk as body, not as a new request/reply